src/System.Net.HttpListener/tests/Configurations.props

```diff
@@ -2,10 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-OSX;
-      netcoreapp-Unix;
-      netcoreapp-Windows_NT;
-      uap-Windows_NT;
+      netstandard-Windows_NT;
```

Comment:
Please also go to the .csproj and modify the project Configurations so that the project still works in VS.